src/java/htsjdk/samtools/cram/ref/ReferenceSource.java

```diff
@@ -63,6 +63,40 @@ public ReferenceSource(final ReferenceSequenceFile rsFile) {
         this.rsFile = rsFile;
     }
 
+    /**
+     * Construct a default ReferenceSource.
```

Comment:
Mention that this is specifically for CRAM, and should only be called when an explicitly-supplied `ReferenceSource` is not available.